packages/flutter/lib/src/material/input_decorator.dart

```diff
@@ -2798,6 +2798,10 @@ class InputDecoration {
   /// of [InputDecoration] that changes the color of the label to the
   /// [InputDecoration.errorStyle] color or [ColorScheme.error].
   ///
+  /// When [InputDecoration.label] is shown above the input field, it is scaled down
+  /// to 0.75. So, any given font size needs to be changed
+  /// accordingly that it will be scaling down at the end.
```

Comment:
@rkishan516 can you help clarify this last sentence? I am not sure what font size you are referring to and how it affects the scaling down in response to focus.

Reply:
Yaa, sure. For any given floatingLabelStyle when we are in focus, we do transform operation to scale down by 75%.
Meaning :- If you are giving fontSize as 24, What you will see is fontSize 18. So, if we want to see fontSize 24, then we should give fontSize as 32.

Reply:
Just FYI, We are not scaling down fontSize rather we are scaling down whole box where its rendering `label`.